media/js/freesound.js

```diff
@@ -154,3 +154,19 @@ function afterDownloadModal(show_modal_url, sound_name){
             openModal();
         });
 }
+function unsecureImageCheck(input) {
+    var unique = '#unsecure_img_check' + new Date().valueOf();
+    var div = $("<div>", {style: "color: red"});
+    div.append("Sorry, images must be from a secure uri.");
```

Comment:
Hmm, yeah ok that's the idea but the message could be more accurate, something like "Warning: you are adding an  image from a non-secure location. Only images from secure locations (i.e. the URL starts with https://...) will be shown in Freesound."